ext/styledmap/eu.esdihumboldt.hale.doc.user.views.styledmap/toc.mapconfiguration.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<toc label="Tasks" link_to="PLUGINS_ROOT/eu.esdihumboldt.hale.doc.user/toc.xml#tasks">
+   <topic href="html/customtile_detail.html" label="Custom Tile Map configuration">
```

Comment:
I think it would be good to add a common parent topic "Map" for both tasks, as they are very map specific and I think it is better not to clog up the main task topic.